### DISCUSSION TITLE
Add bananadroid support

### DISCUSSION THREAD

**Discussion by AbrarNoob** [ROM-builders/temporary#12038] [General]
Previously it was pretty good rom and was supported.. due to some inconvenience it was dead..now alive again..
I created a pr.
https://github.com/ROM-builders/temporary/pull/12037

**Comment by Apon77** [comment]
> Previously it was pretty good rom and was supported.. due to some inconvenience it was dead..now alive again.. I created a pr. #12037

they will be dead again may be.. lets wait for that

**Reply by AbrarNoob** [reply]
Hopefully no..as lead back.. anyway it will be helpful if it's supported..

**Reply by ghost** [reply]
@Apon77 sir add this please as dev is back and source is also buildable and it's good source

**Reply by c0pt4n** [reply]
@Apon77 No bro , there was some inconvenience so it died , now alive again
